roles/api-environment/files/onbase-docs-api/run/Dockerfile

```diff
@@ -1,7 +1,20 @@
-FROM mono:6.6
+FROM mono:latest
 
-WORKDIR /usr/src/$API_NAME
+RUN apt-get update && \
+    apt-get install -y mono-fastcgi-server4 && \
+    apt-get install -y nginx && \
+    touch /run/nginx.pid && \
+    groupadd nginx && \
+    chgrp -R nginx /var/lib/nginx && \
+    chgrp -R nginx /var/log/nginx && \
+    chgrp nginx /run/nginx.pid && \
+    chmod -R 474 /var/lib/nginx && \
+    chmod -R 474 /var/log/nginx && \
+    chmod 474 /run/nginx.pid
```

Comment:
We can do the same trick for `chmod` as well: https://superuser.com/questions/543306/how-can-i-chmod-chown-a-specific-list-of-files